waiter/src/waiter/token.clj

```diff
@@ -206,7 +248,8 @@
           ;;NB do not ever return the password to the user
           (do
             (log/info "successfully retrieved token " token)
-            (utils/map->json-response (merge service-description-template token-metadata)))
+            (utils/map->json-response (merge service-description-template token-metadata)
+                                      :headers {"last-modified" (-> token-metadata (get "last-update-time") str)}))
```

Comment:
The timestamp in the `Last-Modifieid` header also needs to conform:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified

Reply:
Done.